renovate-presets/default.json5

```diff
@@ -90,6 +91,7 @@
     {
       groupName: 'bazel dependencies',
       matchManagers: ['bazel', 'bazel-module'],
+      schedule: ['* 5-12 * * 4'], // 5:00 am am to 12:00 pm Every Thursday
```

Comment:
NIT: consider using a human readable version

Reply:
Interesting, I was not aware that this was deprecated, and usefully deprecations are highlighted in the dashboard/logs.